docs/core/entity/weather.md

```diff
@@ -33,6 +33,8 @@ Forecast data should either be daily or hourly.
 | temperature | float | **Required** | The higher temperature in °C or °F
 | condition | string | `None` | The weather condition at this point.
 | templow | float | `None` | The lower daily Temperature in °C or °F
+| precipitation | int | `None` | The percipitation in mm or inch.
```

Comment:
I believe this does not have to be an integer, floats are fine (and in terms of measuring in inches vital) also any length unit could be used. [Dark Sky for example uses cm for precipitation accumulation](https://darksky.net/dev/docs)

Reply:
Ok but how does a card know which measure to use if they're not part of the config?

Reply:
Fair it has to follow `unit_system` so you’re right mm makes sense to for the metric unit. Basically we need better handling of units wholesale…